client/components/illustration/index.jsx

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const Illustration = ( { illustration } ) => (
+	<div className="illustration-container">
```

Comment:
We can shorten this classname to `illustration`, although I'm not sure we need this component. We do prefer to inline our SVGs over linking as an image